angelsbioprocessing/prototypes/recipes/bio-processing-paper.lua

```diff
@@ -332,7 +336,7 @@ data:extend({
       { type = "item", name = "solid-wood-pulp", amount = 2 },
     },
     results = {
-      { type = "item", name = "solid-paper", amount = 4 },
+      { type = "item", name = "solid-paper", amount = 3 },
     },
     --main_product = "solid-paper",
```

Comment:
Helmod shows the recipes as 'paper', 'paper bleaching 2' and 'paper bleaching 3':
![image](https://github.com/Arch666Angel/mods/assets/26593477/6fd41842-a170-465d-8626-7c064bfd4fd4)

=> to be corrected to 'paper bleaching 1'
=> To be added: `localised_name = {"recipe-name.paper-bleaching-1"},`

For paper 2 and 3 this is not a problem as there are multiple results without a main_product set...

Reply:
![image](https://github.com/Arch666Angel/mods/assets/59639/901bc29d-3f63-424f-97c9-7d2af3ecffeb)

Also fixed similar issue for a few additional recipes.